app/views/pages/moving-guide/claims.html.erb

```diff
@@ -0,0 +1,198 @@
+
+      <p><%= link_to 'For help submitting a Loss & Damage Report, follow this step-by-step tutorial.', page_path('tutorials') %></p>
+
+      <p>
+        <strong>Important Dates:</strong>
+        <br>
+        Submitting a Loss & Damage Report is optional, but more time and flexibility.
+      </p>
+
+      <table>
+        <thead>
+          <tr>
+            <th colspan="2">Important Dates WITH Loss/Damage Report</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th scope="row">Delivery Day (Optional)</th>
+            <td>For any lost or damaged items discovered on delivery day, request that your moving driver complete and sign a DD1840 form. Having mutual agreement on major items, can lead to less disputes when you settle the reimbursement value later on.</td>
+          </tr>
+          <tr>
+            <th scope="row">Up to 30 days after Delivery Date</th>
+            <td>If applicable: Electronically file the DD1840 form created on Delivery Day on the <%= link_to 'Defense Personal Property System (DPS)', 'https://eta.sddc.army.mil/ETASSOPortal/default.aspx' %>.</td>
+          </tr>
+          <tr>
+            <th scope="row">Up to 75 days after Delivery Date</th>
+            <td>Sign-In to the <%= link_to 'Defense Personal Property System (DPS)', 'https://eta.sddc.army.mil/ETASSOPortal/default.aspx' %> and submit a Loss/Damage Report (DD1840R) indicating your intent to file a claim.</td>
+          </tr>
+          <tr>
+            <th scope="row">Up to 9 Months after Delivery Date</th>
+            <td>Sign-In to the <%= link_to 'Defense Personal Property System (DPS)', 'https://eta.sddc.army.mil/ETASSOPortal/default.aspx' %> and submit itemized claims for any lost or damaged items for full replacement value.</td>
+          </tr>
+          <tr>
+            <th scope="row">Between 9 Months and 2 years after Delivery Day</th>
+            <td>Contact the <%= link_to 'Military Claims Office', page_path( 'customer-service', anchor: 'claims') %> for help receiving partial replacement value for any lost or damaged items not previously submitted.</td>
+          </tr>
+          <tr>
+            <th scope="row">60 Days after Claim Submission</th>
+            <td>Your moving company (Transportation Service Provider) should pay, deny, or make an alternate to your claim within 60 days after your claim is final.</td>
```

Comment:
"make an alternate to your claim" Is this what they actually call it? The phrase is awkward to my eyes. This phrase also reappears a few paragraphs down.

Reply:
cc @LillyMadigan @jde1984 @lfantano